src/main/java/net/fabricmc/loom/LoomGradleExtension.java

```diff
@@ -58,6 +58,7 @@
 	public String loaderLaunchMethod;
 	public boolean remapMod = true;
 	public boolean autoGenIDERuns = true;
+	public boolean verbose = true;
```

Comment:
Whats the use of this? Should you just use --info if you want more verbose logging?

We might need to tweak some log outputs to be lower

Reply:
Yeah, I think that right now loom outputs a bit too much info for what it is doing.